shogun-boot/src/main/resources/db/migration/V0.1.0__Baseline.sql

```diff
@@ -1,15 +1,15 @@
 CREATE EXTENSION IF NOT EXISTS postgis WITH SCHEMA public;
 
-SET search_path TO shogun;
+CREATE SCHEMA IF NOT EXISTS shogun;
 
-CREATE SEQUENCE hibernate_sequence
+CREATE SEQUENCE IF NOT EXISTS hibernate_sequence
     INCREMENT 1
     START 1
     MINVALUE 1
     MAXVALUE 9223372036854775807
     CACHE 1;
 
-CREATE TABLE applications (
+CREATE TABLE shogun.applications (
```

Comment:
Migrations must not use a fixed schema, it should be configured by the appropriate jdbc/flyway config.